src/ERC20DelegatesUpgradeable.sol

```diff
@@ -36,39 +41,40 @@ abstract contract ERC20DelegatesUpgradeable is Initializable, ERC20Upgradeable,
 
     /// @custom:storage-location erc7201:morpho.storage.ERC20Delegates
     struct ERC20DelegatesStorage {
-        mapping(address account => address) _delegatee;
+        mapping(address delegator => address) _delegatee;
         mapping(address delegatee => uint256) _delegatedVotingPower;
-        mapping(address account => uint256) _delegationNonce;
+        mapping(address delegator => uint256) _delegationNonce;
     }
 
-    /* PUBLIC */
+    /* GETTERS */
 
-    /// @dev Returns the delegate that `account` has chosen.
-    function delegates(address account) public view returns (address) {
+    /// @dev Returns the delegate that `delegator` has chosen.
+    function delegates(address delegator) public view returns (address) {
         ERC20DelegatesStorage storage $ = _getERC20DelegatesStorage();
-        return $._delegatee[account];
+        return $._delegatee[delegator];
     }
 
-    /* EXTERNAL */
-
-    /// @dev Returns the current amount of votes delegated to `account`.
-    function delegatedVotingPower(address account) external view returns (uint256) {
+    /// @dev Returns the current amount of votes delegated to `delegator`.
+    function delegatedVotingPower(address delegator) external view returns (uint256) {
         ERC20DelegatesStorage storage $ = _getERC20DelegatesStorage();
-        return $._delegatedVotingPower[account];
+        return $._delegatedVotingPower[delegator];
     }
 
-    function delegationNonce(address account) external view returns (uint256) {
+    /// @dev Returns the current delegation nonce of `delegator`.
+    function delegationNonce(address delegator) external view returns (uint256) {
         ERC20DelegatesStorage storage $ = _getERC20DelegatesStorage();
-        return $._delegationNonce[account];
+        return $._delegationNonce[delegator];
     }
 
-    /// @dev Delegates votes from the sender to `delegatee`.
+    /* DELEGATE */
+
+    /// @dev Delegates the balance of the sender to `delegatee`.
     function delegate(address delegatee) external {
-        address account = _msgSender();
-        _delegate(account, delegatee);
+        address delegator = _msgSender();
```

Comment:
Imo it makes comparison more difficult with OZ code, so not sure it's helpful

Reply:
I agree

Reply:
To me we don't have to comply to OZ coding style, and if we find a better (more appropriate for us) way, then we should do it. Here the naming is more specified and can still be consistent, by using delegator/delegatee everywhere, so I find it better